src/pages/taskQueue/TaskQueueTable.tsx

```diff
@@ -36,6 +39,18 @@ export const TaskQueueTable = () => {
     }
   }, [searchChanged, refetchQueue, distro]);
 
+  // SCROLL TO TASK
+  const taskRowRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (taskRowRef.current) {
+      taskRowRef.current.scrollIntoView({
+        behavior: "smooth",
+        block: "start",
+      });
+    }
+  });
```

Comment:
Why is there no dependency array here?
1. Gives console warning if `[taskRowRef.current]` is put as dependency
2. Does not trigger scroll if only `[taskRowRef]` is put as dependency
3. No dependency array = always updates on re-render = 🆗 & no performance hits as far as i can tell